apps/prairielearn/src/tests/accessibility/index.test.ts

```diff
@@ -16,6 +16,8 @@ import * as news_items from '../../news_items/index.js';
 import * as server from '../../server.js';
 import * as helperServer from '../helperServer.js';
 
+import Bootstrap4ConstructPlugin from './bootstrap4-rule.js';
```

Comment:
Nit, consider a different name that matches what's actually being exported.

```suggestion
import Bootstrap4ConstructPlugin from './bootstrap4-construct-plugin.js';
```

Reply:
Done in acb2ae28f88cef2ac4de42e20c492eb8fc19fe99.